src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -81,7 +81,7 @@ public function process(ContainerBuilder $container)
 
                 if (!isset($event['method'])) {
                     $event['method'] = 'on'.preg_replace_callback([
-                        '/(?<=\b)[a-z]/i',
+                        '/(?<=[\b\._])[a-z]/i',
```

Comment:
`\b !== [\b]`

Reply:
I will check it later, but I replaced `\b` with `[\b\._]`, this will match tree elements, which make `kernel.controller_arguments` to be `KernelControllerArguments`, the origin result is `KernelControllerarguments` and the case of word after `_` is wrong